src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -128,6 +128,14 @@ function IOURequestStepConfirmation({
         IOU.setMoneyRequestBillable_temporaryForRefactor(transactionID, defaultBillable);
     }, [transactionID, defaultBillable]);
 
+    useEffect(() => {
+        if (!transaction.category) {
+            return;
+        }
+        if (policyCategories[transaction.category] && !policyCategories[transaction.category].enabled) {
```

Comment:
App crashes on this line

<img width="1261" alt="Screenshot 2024-02-16 at 4 12 39 AM" src="https://github.com/Expensify/App/assets/108292595/8caa50d8-e42e-4f22-b8c0-f8c79ba9d4f9">

Reply:
Thanks. @situchan 

During the first commit of this PR
https://github.com/dukenv0307/App/blob/69e9292b2ec243e5784ea91e9ba5ade2a0fbc24a/src/pages/iou/request/step/IOURequestStepConfirmation.js#L67
`policyCategories` has a default value of `{}` now it has `null`. 🤦 

@dukenv0307 Could you raise a follow-up PR?

Reply:
As no response for 4 days, I raised fix - https://github.com/Expensify/App/pull/36858
This is now 🔥 as app crashes on main user flow and reached production.
@c3024 can you please review?

Reply:
@Gonals @yuwenmemon, the crash was raised while the PR was in staging. It would have been ideal to create a deploy blocker to avoid this crash in production, especially now when we want a stable App version for the earnings call.

@c3024 @situchan, please raise a bug in #expensify-bugs when similar situations happen to have someone create a deploy blocker to avoid crashes into production in the future.

Reply:
Interesting though, that Applause didn't report [the crash](https://github.com/Expensify/App/issues/36854#issue-2144013530) as happening in production 🤷‍♂️

Reply:
yeah, you have to perform the reproduction steps real quick, so maybe it's not a big problem that it is in production, I think it's a rare case